lib/firewall/firewall.cc

```diff
@@ -12,6 +12,25 @@
 #include <timeout.hh>
 #include <vector>
 
+namespace
+{
+#ifdef __LITTLE_ENDIAN__
```

Comment:
I meant in the bodies of these. They should still exist on big endian, they should just return their arguments.